### PR TITLE
Update test data

### PR DESCRIPTION
**Pull request by snnn**
Add:

1. mxnet_arcface
2. tf_mobilenet_v1_1.0_224
3. tf_mobilenet_v2_1.0_224
4. tf_mobilenet_v2_1.4_224
5. tf_inception_v2

#831

### CODE REVIEW

File: onnxruntime/test/onnx/main.cc [hunk 1]
```diff
@@ -339,7 +339,8 @@ int real_main(int argc, char* argv[], OrtEnv** p_env) {
       {"shrink", "test case is wrong"},
       {"maxpool_2d_precomputed_strides", "ShapeInferenceError"},
       {"averagepool_2d_precomputed_strides", "ShapeInferenceError"},
-      {"maxpool_with_argmax_2d_precomputed_strides", "ShapeInferenceError"}
+      {"maxpool_with_argmax_2d_precomputed_strides", "ShapeInferenceError"},
```
Comment (hariharans29) [inline onnxruntime/test/onnx/main.cc]: Doesn't `onnx_backend_test_series.py` require similar changes ?

File: tools/ci_build/github/azure-pipelines/templates/set-test-data-variables-step.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # sets variables $(TestDataUrl) and $(TestDataChecksum)
 
 parameters:
-  TestDataUrl: https://onnxruntimetestdata.blob.core.windows.net/models/20190327.zip
+  TestDataUrl: https://onnxruntimetestdata.blob.core.windows.net/models/20190419.zip
```
Comment (hariharans29) [inline tools/ci_build/github/azure-pipelines/templates/set-test-data-variables-step.yml]: For future reference - is there documentation on how to construct the zip with (potentially) any new models and upload it ?

File: onnxruntime/test/onnx/main.cc (hunk 1 above)
Comment (snnn) [reply onnxruntime/test/onnx/main.cc]: No. onnx_backend_test_series.py doesn't read our test model folder.

File: tools/ci_build/github/azure-pipelines/templates/set-test-data-variables-step.yml (hunk 2 above)
Comment (snnn) [reply tools/ci_build/github/azure-pipelines/templates/set-test-data-variables-step.yml]: We use the same format as onnx model zoo. But, they don't have a doc.

**Comment by snnn** [comment]
@jywu-msft  It's strange that this change causes inception_v2 and densenet fail when mkldnn is enabled. These two models are not new.  see: https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=17046

**Pull request merged**